ydb/core/kqp/executer_actor/kqp_planner.h

```diff
@@ -145,7 +148,8 @@ class TKqpPlanner {
     const TMaybe<size_t> BufferPageAllocSize;
     const bool VerboseMemoryLimitException;
     NScheduler::NHdrf::NDynamic::TQueryPtr Query;
-
+    TActorId CheckpointCoordinatorId;
+    bool CheckpointsReadyStateSended = false;
```

Comment:
Typo `CheckpointsReadyStateSended` -> `CheckpointsReadyStateSent`


```suggestion
    bool CheckpointsReadyStateSent = false;
```